src/components/Comments.js

```diff
@@ -0,0 +1,38 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { BACKEND_URL } from "../constants";
+
+export default function CommentsList() {
+  const { sightingID } = useParams();
+  const [commentsList, setCommentsList] = useState([]);
```

Comment:
idk, this is a bit subjective. But I would name these "commentList", singular instead of plural. Each item is a single comment, and you got a list of it. There are endless debates about naming such variables, feel free to use whatever you prefer :)!